ci/terraform/virtualmachines.tf

```diff
@@ -0,0 +1,41 @@
+resource "harvester_virtualmachine" "cirros-01" {
+  name                 = "cirros"
+  namespace            = "default"
+  restart_after_update = true
+
+  description = "test vm on cirros"
+
+  cpu    = 2
+  memory = "2Gi"
```

Comment:
I'd suggest reducing it to 512 MB since it's a nested VM.

Reply:
Ah, I agree. Thanks for the reminder!!
updated!